### ISSUE TITLE
Revert Integration Test Coverage percentage. Temporarily Lowered it to get a PR in separately from coverage in follow up PR.

### ISSUE THREAD

**Issue by wjuan-AFK** [envoyproxy/nighthawk#564]
PR #563 Was blocked on Test_integration_coverage and I was working on the python tests in a separate PR. So I lowered the threshold in #563 in order to get the process started on reviewing it.

**Comment by dubious90** [comment]
Can you rephrase this bug for me?  The issue appears to be that we are missing integration test coverage.  The lowered coverage percentage is really a side effect of that.  So then we can have one issue (fixed by one PR) where we add in the missing coverage, and raise the coverage percentage back appropriately.  Thanks!

**Comment by oschaaf** [comment]
Is this issue still relevant?

**Comment by mum4k** [comment]
Looks like this isn't relevant anymore. Filed #830 to run a fixit and improve our integration test coverage.
